src/status_im/i18n.cljs

```diff
@@ -52,73 +53,274 @@
+    :enter-address :switch-users :send-transaction :confirmations
+    :recover-access :incorrect-code :image-source-gallery :sync-synced
+    :currency :status-pending :delete-contact :connecting-requires-login
+    :no-hashtags-discovered-title :datetime-day :request-transaction
+    :wallet-send :mute-notifications :invalid-phone :scan-qr :contact-s
+    :unsigned-transaction-expired :status-sending :gas-used
+    :transactions-filter-type :next :recent :confirmation-code
+    :open-on-etherscan :share :status :from :your-current-location
+    :wrong-password :search-chats :transactions-sign-later :in-contacts
+    :transactions-sign :sharing-share :type-a-message :type-a-command
+    :usd-currency :existing-networks :node-unavailable :url :shake-your-phone
+    :add-network :unknown-status-go-error :contacts-group-new-chat :and-you
+    :wallets :clear-history :wallet-choose-from-contacts
+    :signing-phrase-description :no-contacts :here-is-your-signing-phrase
+    :soon :close-app-content :status-sent :status-prompt :testfairy-title
+    :delete-contact-confirmation :datetime-today :add-a-status
+    :web-view-error :notifications-title :error :transactions-sign-transaction
+    :edit-contacts :more :cancel :no-statuses-found :can-not-add-yourself
+    :transaction-description :add-to-contacts :available :open-map
+    :paste-json-as-text :You :main-wallet :process-json :testnet-text
+    :transactions :transactions-unsigned :members :intro-message1
+    :public-chat-user-count :eth :transactions-history :not-implemented
+    :new-contact :datetime-second :search-results :status-failed :is-typing
+    :sharing-copy-to-clipboard-address :recover :wallet-total-value
+    :places-nearby :suggestions-commands :nonce :new-network
+    :contact-already-added :datetime-minute :location
+    :browsing-open-in-web-browser :delete-group-prompt
+    :wallet-insufficient-funds :edit-profile :active-unknown :testfairy-message
+    :search-tags :transaction-failed :public-key :error-processing-json
+    :status-seen :transactions-filter-tokens :status-delivered :profile
+    :wallet-choose-recipient :no-statuses-discovered :none :removed :empty-topic
+    :no :transactions-filter-select-all :transactions-filter-title :message
+    :here-is-your-passphrase :wallet-assets :image-source-title :current-network
+    :left :edit-network-warning :to :data :cost-fee})
+
+;; NOTE: the rest checkpoints are based on the previous one, defined
+;;       like this:
+;; (def checkpoint-2-labels (set/union checkpoint-1-labels #{:foo :bar})
+;; (def checkpoint-3-labels (set/union checkpoint-2-labels #{:baz})
+
+;; NOTE: This defines the scope of each checkpoint. To support a checkpoint,
+;;       change the var `checkpoint-to-consider-locale-supported` a few lines
+;;       below.
+(def checkpoints-def (spec/assert ::checkpoint-defs
+                                  {::checkpoint-0-9-12 checkpoint-0-9-12-labels}))
+(def checkpoints (set (keys checkpoints-def)))
```

Comment:
What happens if new checkpoints are not defined when new keys are added? Any way to list those dangling keys?

Reply:
Nothing happens, that is, tests do not fail, since we decide what keys we want to add to checkpoints.

Just added a function to detect dangling keys: `labels-that-are-not-in-current-checkpoint`.